src/nvim/keycodes.h

```diff
@@ -373,6 +373,9 @@ enum key_extra {
 #define K_KPAGEDOWN     TERMCAP2KEY('K', '5')   // keypad pagedown (lower R.)
 #define K_KORIGIN       TERMCAP2KEY('K', '2')   // keypad center
 
+#define K_KFIND         TERMCAP2KEY('@', '0')   // DEC keys ("Home" and "End"
+#define K_KSELECT       TERMCAP2KEY('*', '6')   // in PuTTY with TERM=xterm)
+
```

Comment:
Why are these placed in the middle of keypad keys? I think these should be placed around `K_HELP`. And why are they named `K_KFIND` and `K_KSELECT` instead of `K_FIND` and `K_SELECT`?

Reply:
> Why are these placed in the middle of keypad keys?

I placed them there because that's where K_HOME and K_END are also defined, even though they're not keypad keys either.

> And why are they named K_KFIND and K_KSELECT instead of K_FIND and K_SELECT?

`K_SELECT` is already in use (by mouse-related things it seems) and I could not decide what other name to choose. Renaming the existing K_SELECT seemed too invasive so I thought "I'll name it K_KSELECT until someone suggests better names".

Reply:
Oh, then I think `K_KFIND` and `K_SELECT` are fine.